packages/reactotron-core-client/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "noImplicitAny": false, // Might be worth setting to true after we get good typing in
+        "removeComments": false,
+        "preserveConstEnums": true,
+        "sourceMap": true,
+        "allowSyntheticDefaultImports": true,
```

Comment:
I don't like that I had to add that but some modules were not wanting to come in.